packages/blockchain-wallet-v4/package.json

```diff
@@ -34,6 +34,15 @@
       "<rootDir>/lib"
     ]
   },
+  "standard": {
```

Comment:
Shouldn't need this since we use ESLint and utilize Jest env support in the root `.eslintrc` file.

Reply:
Also PR #263 adds support for ECMAScript 8 which once merged will fix your lint "violations"

Reply:
Thanks! Will fix